README.md

```diff
@@ -66,7 +67,13 @@ Note: The version tested with Zivid cameras is 1.8.1.
 
        pip install -r requirements.txt
```

Comment:
This currently fails on Ubuntu with Python 3.8.
Is it worth saying something about which Python versions you support?

Reply:
Why does it fail?

Reply:
There is no "pptk" for Python 3.8 yet on PYPI, so "pip install pptk" fails.

See here. Highest is "cp37". No "cp38".
https://pypi.org/project/pptk/#files

Reply:
It looks like PPTK has been dead for two years: https://github.com/heremaps/pptk 
Should consider dropping and replacing, but probably not necessary before merging this.

Reply:
:(

Reply:
But just take a note of it. Let's not delay this. It's really easy to improve this repo incrementally, since there are no official "releases".